source/expressions.tex

```diff
@@ -3332,16 +3339,8 @@
 The class type shall be complete
 unless the class member access appears in the definition of that class.
 \begin{note}
-If the class is incomplete,
-lookup in the complete class type is required to refer
-to the same declaration\iref{basic.scope.class}.
-\end{note}
-The \grammarterm{id-expression} shall name a member of the class or of one of
-its base classes.
-\begin{note}
-Because the name of a class is inserted in its class scope\iref{class},
-the name of a class is also considered a nested
-member of that class.
+The program is ill-formed if the result differs from that
```

Comment:
the "result" of what?

Reply:
The result of the class member access expression.